openshift/onboarding-server/config/owner-addresses.yml

```diff
@@ -0,0 +1,5 @@
+---
+- transport: HTTP
+  port: 80
+  addresses:
+    - dns_name: fido-owner-onboarding.ajo.es
```

Comment:
all of these is definitely duplicated right?

Reply:
This one isn't duplicated, right?, I'm using dns here instead of IP address.

Reply:
the file itlself is duplicated from the root's repo config/ dir right? if so, we can just sed and generate perhaps?

Reply:
I think it's not worth it. I mean, it can be done, of course, and it would remove some small amount of duplicate yaml, but, reading the deployment files (for a human) would be more incomplete.

The other duplicate can be fixed with a simple link.

Anyway, I don't think these yaml files would be something final, more an illustrative example than anything else. I guess an operator or a helm chart would make more sense in the longer term.